src/Microsoft.OData.Client/DataServiceClientFormat.cs

```diff
@@ -4,14 +4,26 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+
+
 namespace Microsoft.OData.Client
 {
+    #region namespaces
     using System;
     using System.Diagnostics;
     using System.Diagnostics.CodeAnalysis;
     using Microsoft.OData;
-    using Microsoft.OData.Edm;
-
+    using Edm;
+    using System.IO;

```

Comment:
Order new ```using namespace``` according to how they have been sorted. In this file, System namespaces come before everything then the others follow.